index.html

```diff
@@ -21,9 +21,8 @@
     <div class='header'></div>
 
     <div class='videoContainer'>
-        <div class='videoView'>
-            <video autoplay poster="static/img/webrtc-logo.png"/>
-        </div>
+        <video class='localVideo' autoplay poster="static/img/webrtc-logo.png"></video>
```

Comment:
you should set muted here to avoid audio feedback, also in the local video CSS flip it horizontally to show a mirrored image